### ISSUE TITLE
[BUG] - Animate Dead token placement freezes map canvas

### ISSUE THREAD

**Issue by Kayne** [otigon/automated-jb2a-animations#666]
**Describe the bug**
When using `Animate Dead` spell, after choosing creature, the gray token appears but cannot be moved and all map canvas is freezed.

The error:
```
Uncaught TypeError: undefined. Cannot set properties of undefined (setting 'renderable')
[Detected 1 package: tokenmagic]
    at e.<anonymous> (settings.js:616:27)
    at a.emit (index.js:202:33)
    at r.dispatchEvent (interaction.min.mjs:8:12235)
    at r.processPointerMove (interaction.min.mjs:8:17255)
    at t.recursiveFindHit (interaction.min.mjs:8:4098)
    at t.findHit (interaction.min.mjs:8:4157)
    at r.processInteractive (interaction.min.mjs:8:12841)
    at r.onPointerMove (interaction.min.mjs:8:16687)
```

Looks like conflict with Token Magic FX module, maybe? I have Token Magic FX on version 0.6.3.1.

**To Reproduce**
1. Use Animate Dead spell
3. Choose Zombie Shambler for instance, but breaks on every creature
4. Freeze!
5. See error in console

**Setup:**
Foundry Version: 10.291
System Version: PF2e 4.12.9
Sequencer Version: 3.1.2
Automated Animations version 4.2.62
Hosting Location: nodejs on server
Browser & version doesn't matter, same behaviour on all

**Active modules:**
Advanced Macros v;
Autocomplete Whisper v;
Automated Animations v;
Award XP v;
Cautious Gamemaster's Pack v;
Chat Images v;
Combat Enhancements v;
Dice So Nice! v;
Dice Tray v;
Drag Ruler v;
Enhanced Terrain Layer v;
Escape Window v;
Foundry VTT Anniversary Dice Set by The Rollsmith v;
Health Estimate v;
Image Hover v;
Initiative Double Click v;
Jam and Jax's NPC Ancestry Abilities v;
Jam and Jax's NPC Index: Specialists v;
Jam and Jax's NPC Index: Spellcaster v;
Jam and Jax's NPC Index: Warriors v;
JB2A - Patreon Complete Collection v;
libWrapper v.0;
LiveKit AVClient v;
Pathfinder 2e: Abomination Vaults v;
Pathfinder Beginner Box v;
Pathfinder Token Pack: Bestiaries v;
Pathmuncher: Pathbuilder character importer for PF2e v;
Perfect Vision v;
PF2e Animations v;
PF2e Companion Compendia v;
PF2e Dorako UI v;
PF2e Drag Ruler Integration v;
PF2E Exploration Effects v;
PF2e Giveth v;
pF2E Keybind Menagerie v;
Pf2E Modifiers Matter v;
PF2e Staves v;
PF2e Target Damage v;
PF2e Workbench v;
Polyglot v;
routinglib v;
Safety Tools v;
Sequencer v;
socketlib v;
Tidy UI - Game Settings v;
Token Action HUD Classic v;
Token Info Icons v;
Tokenizer v;
Token Magic FX v;
Token Mold v;
Warp Gate v;

**Comment by otigon** [comment]
I see nothing to do with Automated Animations in that error message, only Token Magic FX

**Comment by Kayne** [comment]
Right. So I disabled Token Magic FX module and now the placeholder token can be moved and placed on the canvas, however, after all of the summoning animations there is no token art visible. The token is there, tho, it is clickable and moveable. Additionaly greyed out token is visible on starting grid during click-and-drag movement.

<img width="290" alt="Zrzut ekranu 2023-07-30 o 10 38 36" src="https://github.com/otigon/automated-jb2a-animations/assets/1120451/6e9d99d1-b094-471a-a766-7e402019bc20">
<img width="147" alt="Zrzut ekranu 2023-07-30 o 10 39 45" src="https://github.com/otigon/automated-jb2a-animations/assets/1120451/3dbab4e2-a279-41b4-b51b-cec4e1ac1e03">

The art is there. Even when switching token art for different nothing happens. When switched on different tokens, all is working fine (so no "missing picture" issue).

I have also recreated the issue with plain Foundry V11 installation and least viable modules enabled. These are logs since Cast Animate Dead button clicked:

```
commons.js:8713 Foundry VTT | Created MeasuredTemplate with id [bmmGBW3QwMbFxmNI] in parent Scene [EliEFWczsk1w0ZV3]
commons.js:8713 Foundry VTT | Created Token with id [UjtVBB3RLxM4gvxR] in parent Scene [EliEFWczsk1w0ZV3]
commons.js:8713 Foundry VTT | Deleted MeasuredTemplate with id [bmmGBW3QwMbFxmNI] in parent Scene [EliEFWczsk1w0ZV3]
```

The only modules enabled are:
Automated Animations
Foundry Summons
JB2A Patreon
PF2e Animations
Sequencer
socketlib
Warp Gate

All latest versions. Same for foundry VTT (11.306) and pf2e system (5.2.3).
